sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/student-scholastic-standings.institutions.controller.ts

```diff
@@ -196,13 +202,14 @@ export class ScholasticStandingInstitutionsController extends BaseController {
     @UploadedFile() file: Express.Multer.File,
     @Query("validation-only", new DefaultValuePipe(false), ParseBoolPipe)
     validationOnly: boolean,
+    @UserToken() userToken: IInstitutionUserToken,
   ): Promise<PrimaryIdentifierAPIOutDTO[]> {
     // Read the entire file content.
     const fileContent = file.buffer.toString();
-    // Convert the file raw content into text models.
-    let textModels: ApplicationWithdrawalImportTextModel[];
+    // Get the file data.
+    let fileData: FileData;
```

Comment:
I am struggling with the `FileData` as generic as it is.
Maybe it is named something like `BulkWithdrawalFileData` the line 240 would be something like `withdrawalFileData.models,` or even `withdrawalFileData.records` considering that the header is also a model.